src/components/PerfectDropdownSelect/PerfectDropdownSelect.tsx

```diff
@@ -199,14 +208,22 @@ export const PerfectDropdownSelect = <Item extends ItemBase>({
         </Button>
 
         <Divider />
-      </Fragment>
+      </Tooltip>
     ) : (
-      <Button
-        variant='dropdown-element'
-        onClick={() => handleChange(item.value)}
+      <Tooltip
+        content={item.tooltip}
+        align='center'
+        placement='top'
+        disabled={!item.tooltip}
       >
-        {item?.label}
-      </Button>
+        <Button
+          variant='dropdown-element'
+          onClick={() => handleChange(item.value)}
+          isDisabled={item.disabled}
+        >
+          {item?.label}
+        </Button>
+      </Tooltip>
```

Comment:
Sporo kodu dla Tooltipa i Buttona się tu powtarza, zastanawiam się czy nie byłoby dobrze użyć czegoś na wzór [element variables](https://legacy.reactjs.org/docs/conditional-rendering.html#element-variables) + renderowanie childrenów

Reply:
Można by to zoptymalizować.